assets/css/custom.css

```diff
@@ -884,6 +884,38 @@ section.speakers .qcTabTitle .section-title {
   margin: 1em 0;
 }
 
+/* NEWSLETTER SUBSCRIPTION SECTION */
+
+#mailing-list{
+  border: 2px solid #F1D8D3;
+  border-radius: 16px;
+}
+
+.qcWidgetTitle{
+  color: #172751;
+}
+
+#qcWidgets {
+  background-color: #ffffff;
+}
+
+#qcSubscribeWidget input[type=text], input[type=email] {
+  border: 2px solid #F1D8D3;
+  border-radius: 16px;
+  width: auto;
+}
```

Comment:
I think the theme has some width properties on the input. So if you remove `width: auto` it looks good on mobile. So I would suggest using a media query: 
``` css
#qcSubscribeWidget input[type=text], input[type=email] {
  border: 2px solid #F1D8D3;
  border-radius: 16px;
}

@media screen and (min-width: 768px) {
  #qcSubscribeWidget input[type=text], input[type=email] {
    width: auto;
  }
}
```